modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/Godot.SourceGenerators.csproj

```diff
@@ -21,8 +21,7 @@
     <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.8.0" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.3" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.8.0" PrivateAssets="all" />
```

Comment:
Can you explain why this change is needed? I suspect this is causing the `CS8032` diagnostic that @paulloz is getting.

Reply:
Thankfully not an expert in that.
I was thinking that, there are high-level packages like "Microsoft.CodeAnalysis.CSharp.Workspaces" or "Microsoft.CodeAnalysis.CSharp.Features". Any of those two references all the required smaller packages.
 
 When implementing a code-fix, I was looking at https://github.com/microsoft/Microsoft.Unity.Analyzers/blob/9159bac481bfcfe01913c4ac9594b1fdcb139734/src/Microsoft.Unity.Analyzers/Microsoft.Unity.Analyzers.csproj#L12 as a reference.

Reply:
downgrading the version seem to be fine. pushed that.